marklogic-data-hub/src/main/resources/ml-config/database-fields/final-database.xml

```diff
@@ -114,5 +114,12 @@
       <range-value-positions>true</range-value-positions>
       <invalid-values>reject</invalid-values>
     </range-path-index>
+    <range-path-index>
```

Comment:
HubProjectImpl.upgradeProject will need to be modified to inject this index into the user's existing final-database.xml file. Odds are, the user hasn't modified it, but we don't want to overwrite the file in case the user has made modifications. The contents of the file can be read in as an XML document, and then you can just manually add this new index to it, and then write it back to disk.

Reply:
Modified HubProjectImpl.upgradeProject to inject the new index